test/utils/retry-until.ts

```diff
@@ -0,0 +1,5 @@
+import { wait } from './wait'
+
+export const retryUntil = ( fn: () => Promise<void> ) => fn()
+  .catch( () => wait( 1000 ) )
```

Comment:
Why wait?

Reply:
Because if you're using this for something like a filesystem operation, you don't want to hammer the filesystem by instantly retrying, you want to wait first. If the `.catch()` reutrns a promise, which it does, it becomes chained by the `.then`, which will execute.

But I actually do see a problem with the code in that it may always constantly retry if successful, will double check...